lua/ttt2/extensions/net.lua

```diff
@@ -0,0 +1,99 @@
+---
+-- net stream extension
+-- @author saibotk
+
+AddCSLuaFile()
+
+-- Stream network message name constant
+local NETMSG_STREAM = "TTT2_NET_STREAM"
+
+-- Size to split the network stream at (currently a bit lower than the max value, just to have some buffer)
+net.STREAM_FRAGMENTATION_SIZE = 65400
```

Comment:
Could you shortly explain why using 65400 as limit not 65535 (minus default send bits)?

Reply:
Because i wanted to have a small buffer, as calculating this number seemed to be a bit of a good guess and i did not want to have this break, just because of some net function that could change its size in the future.